spaceship/lib/spaceship/tunes/application.rb

```diff
@@ -120,11 +130,48 @@ def resolution_center
         client.get_resolution_center(apple_id, platform)
       end
 
+      def platforms
+        platforms = []
+        version_sets.each do |version_set|
+          platforms << version_set.platform
+        end
+        platforms
+      end
+
+      def types
```

Comment:
Line 172 is:
`elsif self.platforms == %w(ios appletvos)`
Not seeing a call to `types` here, hmm...